android/java/org/chromium/chrome/browser/crypto_wallet/util/TokenUtils.java

```diff
@@ -131,7 +131,7 @@ public static void getAllTokens(
         blockchainRegistry.getAllTokens(
                 chainId,
                 coinType,
-                tokens -> callback.call(Utils.fixupTokensRegistry(tokens, chainId)));
```

Comment:
The method `TokenUtils.getAllTokens` is now useless as it immediately calls `BlockchainRegistry.getAllTokens`.
We should completely remove this method and replace its occurrences with  `blockchainRegistry.getAllTokens`.

Reply:
Fixed here: https://github.com/brave/brave-core/pull/24438/commits/cd9e87f7eb2a50bdd4370a7c131a39ccbfcfa134